<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="retriveTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>991</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tableTab">
    <attribute name="title">
     <string>Table</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>961</width>
       <height>471</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Temperature(Front)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temperature(Rear)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Heat (Left Seat)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Heat (Right Seat)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fan Speed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temperature (Ambient)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>A/C Active</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Recirculation Active</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Created at</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="tablePopulate">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>490</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Retrieve</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="graphTab">
    <attribute name="title">
     <string>Graph</string>
    </attribute>
    <widget class="QCustomPlot" name="graphView" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>831</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>290</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="graphList"/>
      </item>
      <item>
       <widget class="QPushButton" name="graphPopulate">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="graphRetrieve">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>240</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Retrieve</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="results">
      <property name="text">
       <string>Results</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="numResults">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
